marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.smart-mastering/survivorship/merging/base.xqy

```diff
@@ -53,7 +53,10 @@ import module namespace util-impl = "http://marklogic.com/smart-mastering/util-i
   at "/com.marklogic.smart-mastering/impl/util.xqy";
 import module namespace mem = "http://maxdewpoint.blogspot.com/memory-operations/functional"
   at "/mlpm_modules/XQuery-XML-Memory-Operations/memory-operations-functional.xqy";
-import module namespace es-helper = "http://marklogic.com/smart-mastering/entity-services" at "/com.marklogic.smart-mastering/sm-entity-services.xqy";
+import module namespace es-helper = "http://marklogic.com/smart-mastering/entity-services"
+  at "/com.marklogic.smart-mastering/sm-entity-services.xqy";
+import module namespace helper = "http://marklogic.com/smart-mastering/helper-impl"
```

Comment:
"helper-impl" is used as a prefix above - I find it's best to use the same prefix throughout the codebase to make it easier to find all references to a function via global search. I think "helper" is the appropriate prefix, unless there's an existing convention in the SM code for using "(something)-impl" for impl libraries.